extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia.com>
+ * @version: $Id$
+ */
+
+use Wikia\Service\User\Permissions\PermissionsServiceAccessor;
+
+class ChatBanData
+{
+	use PermissionsServiceAccessor;
+
+	/**
+	 * @var string
+	 */
+	private $cityId;
+
+	/**
+	 * @var string
+	 */
+	private $userName;
+
+	/**
+	 * @var int
+	 */
+	private $limit;
+
+	/**
+	 * @var int
+	 */
+	private $offset;
+
+	/**
+	 * @var string
+	 */
+	private $order;
+
+	/**
+	 * @var array
+	 */
+	private $orderOptions;
+
+	/**
+	 * @var Object
+	 */
+	private $skin;
+
+	/**
+	 * Database
+	 * @var string
+	 */
+	private $db;
+
+	/**
+	 * @var string
+	 */
+	private $table;
+
+
+	function __construct( $city_id, $load = 1 ) {
+		global $wgExternalDatawareDB;
+
+		$this->cityId = $city_id;
+		$this->db = $wgExternalDatawareDB;
+		$this->table = 'chat_ban_users';
+		$this->skin = RequestContext::getMain()->getskin();
+
+		$this->orderOptions = [
+			'timestamp' => 'start_date %s',
+			'target'    => 'cbu_user_id %s',
+			'expires'   => 'end_date %s',
+			'blockedBy' => 'cbu_admin_user_id %s',
+			'reason'    => 'reason %s',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "reason" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)